generator/support.go

```diff
@@ -170,15 +170,16 @@ func checkPrefixAndFetchRelativePath(childpath string, parentpath string) (bool,
 
 }
 
+// TODO: handle error with a returned error rather than panic()
 func baseImport(tgt string) string {
 	tgtAbsPath, err := filepath.Abs(tgt)
 	if err != nil {
-		log.Fatalln(err)
+		log.Fatalln("could not evaluate base import path with target \"%s\". Target directory must be created beforehand: %v", tgt, err)
```

Comment:
you probably mean log.Fatalf here
ping me in slack so we can walk through getting an env where the editor you use tells you about these stylistic errors

Reply:
Yes, you're probably right I haven't looked hard enough into vim add-ons.
Don't merge the PR yet: a number of unit test is coming up.
I let the CI run to figure out the coverage.